Linkta-org/core
# Add cropped logo with no background

## Description

This PR adds a cropped version of the logo and removes the background



Fixes # (issue)

## Type of change




## How Can this be tested? Testing Plan to review this PR


The logo changed is the original logo on the logged in page. 
- Pull down branch, log in, and see new cropped logo in main layout.  


## Checklist:




## Additional Information:
